conda_smithy/configure_feedstock.py

```diff
@@ -385,7 +384,28 @@ def compute_build_matrix(meta, existing_matrix=None):
     return mtx
 
 
-def main(forge_file_directory):
+def write_rendered_file(target_fname, template, forge_config, strict=False):
+    content = template.render(**forge_config)
+    if strict and (os.path.isfile(target_fname)):
+        with io.open(target_fname, 'r', encoding='utf-8') as fh:
+            old_content = fh.readlines()
+        
+        # If the existing file has exactly one changed line (len(diff) == 2)
```

Comment:
This comment seems to be outdated - we really are looking at the number of lines that have changed that aren't proceeded by a hash.